account_document_reversal/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Account Document Reversal",
+    "summary": "Create reversed journal entries when cancel document",
+    "version": "13.0.1.0.0",
+    "author": "Ecosoft," "Eficent," "Odoo Community Association (OCA)",
```

Comment:
@kittiu can you change Eficent -> ForgeFlow?

Reply:
Done